src/http/content_encoding.rs

```diff
@@ -7,16 +7,15 @@
 // except according to those terms.
 
 use std::cmp::Ordering;
-use std::io::{self, BufReader};
+use std::io::{self, Read};
```

Comment:
Nope. I just personally don't like `use std::io::prelude::*` because I cannot predict what will be imported. However, I don't think importing `Read` trait on mod scope instead of function scope is better. I decide to revert on 481b1fb.

Reply:
You're correct. We don't need to import the whole IO prelude. Thank you for the explanation!